src/uct/ib/mlx5/ib_mlx5_log.c

```diff
@@ -449,6 +419,49 @@ void uct_ib_mlx5_cqe_dump(const char *file, int line, const char *function, stru
     uct_log_data(file, line, function, buf);
 }
 
+void uct_ib_mlx5_av_dump(char *buf, size_t max,
+                         uint32_t dqp_dct, uint16_t rlid, /* base AV */
```

Comment:
maybe pass ptr to base_av struct?

Reply:
@yosefe yes, it would be better, but UCT/IB/UD/MLX5 couldn't provide it from EP:
https://github.com/openucx/ucx/blob/55b70af6ae4c021de2114fbaab34a26da01ba326/src/uct/ib/ud/accel/ud_mlx5.h#L16

Reply:
then pass uct_ib_mlx5_base_av_t* to uct_ib_mlx5_av_dump?
uct_ib_mlx5_dump_dgram could fill it as local var

Reply:
@yosefe done